packages/blaze/template.js

```diff
@@ -309,6 +311,43 @@ Blaze.TemplateInstance.prototype.autorun = function (f) {
 };
 
 /**
+ * @summary A version of [Meteor.subscribe](#meteor_subscribe) that is stopped
+ * when the template is destroyed.
+ * @return {SubscriptionHandle} The subscription handle to the newly made
+ * subscription. Call `handle.stop()` to manually stop the subscription, or
+ * `handle.ready()` to find out if this particular subscription has loaded all
+ * of its inital data.
+ * @locus Client
+ * @param {String} name Name of the subscription.  Matches the name of the
+ * server's `publish()` call.
+ * @param {Any} [arg1,arg2...] Optional arguments passed to publisher function
+ * on server.
+ * @param {Function|Object} [callbacks] Optional. May include `onError` and
+ * `onReady` callbacks. If a function is passed instead of an object, it is
+ * interpreted as an `onReady` callback.
+ */
+Blaze.TemplateInstance.prototype.subscribe = function (/* arguments */) {
+  var subHandle = this.view.subscribe.apply(this.view, arguments);
+
+  this._subscriptionHandles.set(
+    this._subscriptionHandles.get().concat([subHandle]));
+
+  return subHandle;
+};
+
+/**
+ * @summary A reactive function that returns true when all of the subscriptions
+ * called with [this.subscribe](#TemplateInstance-subscribe) are @ready.
+ * @return {Boolean} True if all subscriptions on this template instance are
+ * ready.
+ */
+Blaze.TemplateInstance.prototype.ready = function () {
```

Comment:
And we should mention it wherever we mention breaking changes.

It could turn out that a lot of people set `this.ready` in their apps and are unclear on whether they need to change that code now or not.